test/verify/quant_dot_3args_3.cpp

```diff
@@ -10,9 +10,9 @@ struct quant_dot_3args_3 : verify_program<quant_dot_3args_3>
     {
         migraphx::program p;
         auto* mm = p.get_main_module();
-        migraphx::shape m1_shape{migraphx::shape::int8_type, {2, 8}};
+        migraphx::shape m1_shape{migraphx::shape::int8_type, {4, 8}};
         migraphx::shape m2_shape{migraphx::shape::int8_type, {7, 8}};
-        migraphx::shape m3_shape{migraphx::shape::int32_type, {2, 7}};
+        migraphx::shape m3_shape{migraphx::shape::int32_type, {4, 7}};
```

Comment:
Why do we change this to `4`? We should have tests for <4 matrices.

Reply:
If less than 4, and the QA test our code on an MI100, this unit test will fail.

Reply:
The plan is: when rocm4.4 is available and hopefully less than 4 cases are supposed, we will add these tests back. 
I created an issue #835 for tack this problem.

Reply:
Alright, well can we just disable the test for the gpu instead of changing it? You can disable tests for the gpu in test/verify/main.cpp.

Reply:
fixed